src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs

```diff
@@ -25,6 +25,8 @@ public class GenevaExporterOptions
 
     public EventNameExportMode EventNameExportMode { get; set; }
 
+    public bool IncludeTraceState { get; set; }
```

Comment:
```suggestion
    public bool IncludeTraceStateForSpan { get; set; }
```

something to indicate this is only for Spans, not LogRecords..

Reply:
Would `IncludeTraceStateForTrace` be a better name?

Reply:
"TraceStateForTrace" might be a little tongue twisting...

Reply:
Renamed to `IncludeTraceStateForSpan`.